docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -74,14 +74,51 @@ bigquery.project-id=<your Google Cloud Platform project id>
 
 ### Multiple GCP projects
 
-The BigQuery connector can only access a single GCP project.Thus, if you have
+The BigQuery connector can only access a single GCP project. Thus, if you have
 data in multiple GCP projects, You need to create several catalogs, each
 pointing to a different GCP project. For example, if you have two GCP projects,
 one for the sales and one for analytics, you can create two properties files in
 `etc/catalog` named `sales.properties` and `analytics.properties`, both
 having `connector.name=bigquery` but with different `project-id`. This will
 create the two catalogs, `sales` and `analytics` respectively.
 
+(bigquery-project-id-resolution)=
+### Project ID resolution
+
+The BigQuery connector can be configured to use different project for billing, 
+see more in the 
+[Google Cloud documentation](https://cloud.google.com/iam/docs/attach-service-accounts).
+The BigQuery connector determines the project ID to use based on the 
+configuration settings. The following table explains how project IDs 
+are resolved in different scenarios:
+
+:::{list-table} Project ID resolution
+:widths: 30, 33, 33
+:header-rows: 1
+
+* - Configured properties
+  - Billing project in BigQuery
+  - Data querying project ID
+* - Only `bigquery.credentials-key`
+  - The project ID from the service account in the credentials key 
```

Comment:
nit: here we say "project ID from the service account in the credentials key" and some places we say "project ID from the credentials key". Let's keep the latter everywhere (shorter and no risk of confusion).

Reply:
Applied!